diagnostic.md

```diff
@@ -73,14 +88,20 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request POST http://www.example.com/donuts ("French Cruller", "$"+"0.99")
 ```
 
 Write an AJAX request to create a single donut on the server using JSON. Please
 do not use `data = getFormFields(form)` instead write out the data object.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function (data) {
+  return $.ajax({
+    url: app.host + '/donuts/',
+    method: 'POST',
+    data: data
```

Comment:
They want you to write out the data object here, but I get what you're going for